North-Seattle-College/ad440-winter2022-thursday-repo
# Feature - Style page frontend

Closes #95 - Style the page frontend according to the Floop styleguide

**Description of Change**
- Added Floop's logo
  - https://app.floopedu.com/PressKit
- Renamed the comment/feedback box headers

- Styled the page using the below style guide 

![image (1)](https://user-images.githubusercontent.com/47171828/156041751-a11e3e8e-6331-4a64-9930-e7e4b42410f0.png)

**Updated page frontend:**

<img width="1440" alt="Screen Shot 2022-02-28 at 11 09 45 AM" src="https://user-images.githubusercontent.com/47171828/156044053-264ebba2-35b2-4c7f-874f-9bc47a34fe24.png">

**How to test**
1. Clone the repo
2. cd frontend 
3. npm install
4. npm start
5. git checkout feature-style-page
6. The page's frontend will appear with all the new updates
